package.json

```diff
@@ -37,7 +37,6 @@
     "ajv": "^8.12.0",
     "ajv-errors": "^3.0.0",
     "axios": "^0.21.1",
-    "bignumber.js": "^9.0.1",
```

Comment:
```package-lock.json``` also should be updated.
make sure that ethereum-cryptography package is fine and there is no error after updating package-lock